packages/nova-base-components/lib/common/NewsletterButton.jsx

```diff
@@ -9,12 +9,11 @@ class NewsletterButton extends Component {
     super(props);
     this.subscriptionAction = this.subscriptionAction.bind(this);
   }
-  
+
   subscriptionAction() {
```

Comment:
`props.user` is needed here to handle a tricky edge case: you are someone that has permission to edit users & you are editing someone profile, the subscribe button should act on the user you are editing and not yourself.

...📬 🌮 

Another solution could to simply show the newsletter custom field if you are editing your own profile.